trismart_smartboard_integration/models/smartboard_connector.py

```diff
@@ -59,9 +70,9 @@ def create_project(self, sb_lead_id, x_api_key, project_template_id=None):
                         project = template.create_project_from_template(sb_lead_id)
                         project_new = project_env.search([('id', '=', project['res_id'])], limit=1)
                     else:
-                        project_new = project_env.create({"name": 'SBLead-' + str(sb_lead_id), "sb_lead_id": sb_lead_id})
+                        project_new = self.create_non_template_project(sb_lead_id)
                 else:
-                    project_new = project_env.create({"name": 'SBLead-' + str(sb_lead_id), "sb_lead_id": sb_lead_id})
+                    project_new = self.create_non_template_project(sb_lead_id)
```

Comment:
Can we create_non_template_project only once ? Is there any way to merge 2 cases into one since both cases do exactly the same thing?